src/vm/arm/exceparm.cpp

```diff
@@ -54,6 +54,35 @@ FaultingExceptionFrame *GetFrameFromRedirectedStubStackFrame (T_DISPATCHER_CONTE
     return (FaultingExceptionFrame*)((TADDR)pDispatcherContext->ContextRecord->R4);
 }
 
+//Return TRUE if pContext->Pc is in DispatchStub
+BOOL IsDispatchStub(CONTEXT *pContext)
```

Comment:
Actually, if we would support both virtual stubs, I think that we should include the tests for DISPATCH_STUB_FIRST_WORD and RESOLVE_STUB_FIRST_WORD here, name the function IsIPInVirtualStubFirstWord and modify AdjustContextForVirtualStub to call it instead of repeating the code in both functions.